src/tools/emscripten.jam

```diff
@@ -94,12 +120,13 @@ actions compile.c++
 
 actions archive
 {
-    "$(CONFIG_COMMAND)" $(AROPTIONS) -r -o "$(<)" "$(>)"
+    "$(.AR)" $(AROPTIONS) rc "$(<)" "$(>)"
+    "$(.RANLIB)" "$(<)"
 }
 
 toolset.flags emscripten.link USER_OPTIONS <linkflags> ;
 
 actions link bind LIBRARIES
 {
-    "$(CONFIG_COMMAND)" $(USER_OPTIONS) -L"$(LINKPATH)" -o "$(<)" "$(>)" "$(LIBRARIES)" $(START-GROUP) $(FINDLIBS-ST-PFX) -l$(FINDLIBS-ST) $(FINDLIBS-SA-PFX) -l$(FINDLIBS-SA) $(END-GROUP) $(OPTIONS)
+    "$(CONFIG_COMMAND)" $(USER_OPTIONS) -L"$(LINKPATH)" -o "$(<)" "$(>)" "$(LIBRARIES)" $(START-GROUP) -l$(FINDLIBS-ST) -l$(FINDLIBS-SA) $(END-GROUP) $(OPTIONS)
```

Comment:
I think you need to modify GCC instead https://github.com/bfgroup/b2/blob/371b47af9dc837624d7af50906653074f8ea1475/src/tools/gcc.jam#L933-L941

Reply:
Can these settings be overridden in emscripten.jam?

Reply:
Will the following in emscripten.jam do the job?

```
    toolset.flags emscripten.link FINDLIBS-ST-PFX <target-os>web/<runtime-link>static : ;
    toolset.flags emscripten.link FINDLIBS-SA-PFX <target-os>web/<runtime-link>static : ;
```

Anyway, I will try.